packages/nextjs/contracts/deployedContracts.ts

```diff
@@ -4,6 +4,335 @@
  */
 import { GenericContractsDeclaration } from "~~/utils/scaffold-eth/contract";
 
-const deployedContracts = {} as const;
```

Comment:
hey!! i am actually confused about this file. does changing the chain to optimism and redeploying fixes the issue? it seems like the file was auto-generated when i was testing on localhost. could you help me understand it better?

Reply:
Once you deploy a contract to a chain, it is automatically created. This is useful when deploying a new contract to the mainnet, as it allows other contributors to know which external contracts are being used, along with their respective addresses. On a local network, however, the contract addresses can vary since everyone deploys their own instances. Since we're focusing on front-end work and not working on new smart contracts here, we'd prefer not to modify the file. 

Hopefully that makes sense, pls let me know.

Reply:
alright! i got it. made changes to undo the modifications to this file. will keep this in mind next time. thanks!